initial-configuration/uninstall.sh

```diff
@@ -17,7 +17,8 @@ images=$(docker ps -a -q)
 if [ ! -z "$images" ]; then
 	docker rmi $(docker images -a -q)
 fi
-docker system prune -a
+docker system prune -a --volumes -f
+docker volume prune -a -f
```

Comment:
```suggestion
docker system prune -a --volumes -f
docker volume prune -a -f
```

verify if this is necessary after I fix the mysql mount